src/__experimental__/components/radio-button/radio-button.component.js

```diff
@@ -19,7 +19,6 @@ const RadioButton = ({
     ...props,
     onChange: handleChange,
     onBlur,
-    helpTabIndex: 0,
     helpTag: 'span',
```

Comment:
Unrelated but it looks like this can be removed, it's only defined once. But looks like it'd be a breaking change 👎